Uni_life_tracker/app/src/main/java/Entities/StudySession.java

```diff
@@ -1,25 +1,37 @@
-package Entities;/*
-NOTE: TO BE IMPLEMENTED LATER IN PHASE 1!!
-*/
+package Entities;
+
+import androidx.annotation.NonNull;
+
 import java.util.ArrayList;
+import java.time.LocalDateTime;
 
-public class StudySession {
+/**
+ * Represents a study session of a course, with participants, location and a note.
+ */
+public class StudySession extends Event {
     private String course;
     private String location;
     private String notes;
```

Comment:
I think we decided that notes was no longer an instance variable for any of the event types.

Reply:
I actually thought about removing it. I'll get working on that.